src/bundle/Resources/public/js/scripts/admin.content.edit.js

```diff
@@ -54,6 +54,12 @@
     };
 
     form.setAttribute('novalidate', true);
+    form.onkeypress = (e) => {
+        let key = e.charCode || e.keyCode || 0;
+        if (key == 13) {
```

Comment:
```suggestion
        if (keyCode === 13) {
```

Please define `13` as a meaningfull variable